### ISSUE TITLE
Link para a nossa wiki

### ISSUE THREAD

**Issue by barrosgg** [womozbrasil/womoz.mozillabrasil.org.br#1]
Hey galera ;-)

É necessário adicionar o link[1] da Wiki WoMoz Brasil no menu do site, isso é importante pois a wiki possuí uma documentação elaborada e fundamentada acerca de nossa luta, além de ser um importante portal de transparência do projeto.

[1] https://wiki.mozilla.org/Brasil/WoMoz

**Comment by halcance** [comment]
@barrosgeraldo sim, seria ótimo mas ainda não entendi como funciona  o site, @cynthiapereira ele está hospedado no github ou no wordpress ? Se for no wordpress creio que não poderei fazer isso.
